static/js/i18n/hi.json

```diff
@@ -0,0 +1,21 @@
+{
+  "page-title": "Google Code-in 2017 वर्तमान नेता",
+  "last-updated": "आखरी अपडेट",
```

Comment:
where did you get this translation from?

Is that translation MIT licensed per https://github.com/coala/gci-leaders/issues/6?
If not, please remove, and we'll acquire the translation again using a method that complies with the intended license of this repo.

Reply:
IMO we should remove all translation for now and only keep `en` or 2.